code/Model/DynamoDbSession.php

```diff
@@ -92,8 +98,9 @@ public function __construct($options, $table)
      */
     protected function getSessionLifetime()
     {
-        if (defined('AWS_DYNAMODB_SESSION_LIFETIME')) {
-            return AWS_DYNAMODB_SESSION_LIFETIME;
+        $AWS_DYNAMODB_SESSION_LIFETIME = getenv('AWS_DYNAMODB_SESSION_LIFETIME');
+        if (!empty($AWS_DYNAMODB_SESSION_LIFETIME)) {
+            return $AWS_DYNAMODB_SESSION_LIFETIME;
```

Comment:
As discussed, it would be good if we could add some tests, but that might not be part of this PR